plugins/table_view/web_client/stylesheets/tableWidget.styl

```diff
@@ -0,0 +1,18 @@
+.g-item-table-view-header
+  background-color #f0f0f0
+  padding 5px 6px
+  color #555
+  font-size 16px
+  font-weight bold
+  margin-top 18px
+  cursor pointer
+  display flex
+
+  .g-item-table-view-title
+    flex 1
+
+.g-item-table-view-container
+
+  table
+    display block
+    overflow scroll
```

Comment:
I'd recommend adding something like
```
li.g-table-view-page-next,li.g-table-view-page-prev
  user-select none
```
so that one doesn't select (and highlight) the text of the buttons.  I found that clicking repeatedly would end up blink the highlight on the button text otherwise.  This gets added to `.btn` classes automatically, so the other way would be to change the css class.